styles/styles.css

```diff
@@ -26,12 +26,15 @@
   --heading-font-family: 'HCo Ringside Extra Wide', 'HCo Ringside Extra Wide Fallback', sans-serif;
```

Comment:
some of these styles are not yet used in this PR but are referenced already in my working copy, sorry about that.